### PR TITLE
BZ-2097619-4-12 update vSphere driver version in 4.12

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
This is one of three PRs for this BZ, each documenting a different version of a third-party storage driver specific to each major OpenShift release.

Version(s):
PR applies to: 4.12

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2097619

Link to docs preview:
https://51371--docspreview.netlify.app/openshift-enterprise/latest/storage/container_storage_interface/persistent-storage-csi-vsphere.html#overview

Additional information:
https://github.com/openshift/openshift-docs/pull/51384
https://github.com/openshift/openshift-docs/pull/51385

### CODE REVIEW

**Review by kcarmichael08 — COMMENTED** [review]
I believe you need to use the product-version variable (I'm checking with Kathryn to make sure). Otherwise, LGTM! (Edit - yep, you need to use the variable.)

File: storage/container_storage_interface/persistent-storage-csi-vsphere.adoc [hunk 1]
```diff
@@ -16,7 +16,11 @@ To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage
 
 * *vSphere CSI Driver Operator*: The Operator provides a storage class, called `thin-csi`, that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
 
-* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs.
+* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs. In {product-title} 4.12, the driver version is 2.6.1.
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply storage/container_storage_interface/persistent-storage-csi-vsphere.adoc:19]: For this specific case, I chose to manually write it in so nothing slips. The driver is 1-to-1 with the OCP version and doesn't change, per dev.

**Pull request merged**